apollo-client/src/main/java/com/ctrip/framework/apollo/spring/annotation/ApolloConfig.java

```diff
@@ -49,6 +49,10 @@
 @Target(ElementType.FIELD)
 @Documented
 public @interface ApolloConfig {
+  /**
+   * Apollo appId for the config, if not specified then default to application
+   */
+  String appId() default "";
```

Comment:
```suggestion
  /**
   * Apollo appId for the config, if not specified then default to the global appid
   *
   * @since 2.4.0
   */
  String appId() default "";
```